### PR TITLE
silence join event and reduce warmup end time.

### PR DESCRIPTION
**Pull request by SurajBhari**
warmup restarts multiple times when map changes with multiple people in it. 
this somehow fixes it (i swear i forgot the reasoning how it works) but it works.

### CODE REVIEW

File: scripting/executes.sp [hunk 1]
```diff
@@ -654,7 +654,7 @@ public void EndWarmup() {
                         g_hMinPlayers.IntValue);
   ServerCommand("mp_death_drop_gun 1");
   ServerCommand("mp_warmup_pausetimer 0");
-  ServerCommand("mp_warmuptime 10");
+  ServerCommand("mp_warmuptime 3");
```
Comment (splewis) [inline scripting/executes.sp]: Is this change necessary to fix the issue, or is this a style/gameplay choice? IMO, 3 seconds is a bit short.

Comment (SurajBhari) [reply scripting/executes.sp]: I think the main reason around this bug is a race condtion. we are just escaping that race condition tbh. 
and i always wondered why you force some commands. specially warmuptime . isn't it better to make them added in a cfg rather than forced by a plugin?

Comment (splewis) [reply scripting/executes.sp]: > isn't it better to make them added in a cfg rather than forced by a plugin?

Yes. It's just more overhead and configuration that can become wrong (e.g. file is missing, user adds commands that triggers some bug), so I didn't bother here.

Comment (SurajBhari) [reply scripting/executes.sp]: well in that case. i guess this can be changed to 5 seconds. and then its all good to merge ?. i prefer 3 seconds personally.

**Comment by SurajBhari** [comment]
Waiting for a review.
